docs/current_docs/manuals/developer/cookbook/snippets/error-handling/go/main.go

```diff
@@ -0,0 +1,24 @@
+package main
+
+import (
+	"context"
+	"fmt"
+
+	"dagger.io/dagger/dag"
+)
+
+type MyModule struct{}
+
+// Generate an error
+func (m *MyModule) Test(ctx context.Context) (string, error) {
+	out, err := dag.
+		Container().
+		From("alpine").
+		// ERROR: cat: read error: Is a directory
+		WithExec([]string{"cat", "/"}).
+		Stdout(ctx)
+	if err != nil {
+		return "", fmt.Errorf("Test pipeline failure: %w", err)
```

Comment:
@marcosnils @kpenfound can you help me modify this code so that it returns the same `stderr` output as in the TS and Python snippets? I'm looking for `Test pipeline failure: cat: read error: Is a directory`